Common/src/gRPC/Services/GrpcResultsService.cs

```diff
@@ -156,7 +156,7 @@ public class GrpcResultsService : Results.ResultsBase
                                                                          Guid.NewGuid()
                                                                              .ToString(),
                                                                          rc.Name,
-                                                                         "",
+                                                                         request.SessionId,
```

Comment:
is it normal that SessionId is passed two times :one for SessionId and another for OwnerTaskId?

Reply:
Yes that is the purpose of this pr